### DISCUSSION TITLE
Does it stream sound?

### DISCUSSION THREAD

**Discussion by adRn-s** [browsh-org/browsh#482] [Q&A] [answered]
I assume it doesn't. The gif at the readme "sounded" misleading :P

If I'm correct, then the real question is: Can't it be extended to stream sound too?

**Comment by tombh** [comment, marked as answer]
It only streams sound if Firefox is in the same X session, which isn't the intended usecase of Browsh. So the answer is no I'm afraid.

**Reply by adRn-s** [reply]
Ah, I see. So there's no way to bypass such a technical limitation. Adding an optional parallel VNC connection **only for streaming the sound** would be too difficult to implement, and or defeats the bandwidth saving, right? Great tool nonetheless. Thanks!
